### ISSUE TITLE
Device/Save As does not update current device (Trac #133)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#137]
Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/133
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Device/Save As does not update current device", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-11-01T15:48:23", 
    "milestone": "Microdrop 1.0", 
    "owner": "", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20140320T13:56:23: 
* cfobel commented:

> Fixed as of 030f94d1b727b19b2d70b9d402d5569b445b8032 //(currently only on the `dev` branch)//.


* cfobel changed resolution from "" to "fixed"
* cfobel changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "" to "microdrop"
